ethsigner/signer/multiplatform/src/test/java/tech/pegasys/ethsigner/signer/multiplatform/FileBasedSigningMetadataFileTest.java

```diff
@@ -29,7 +31,7 @@
   @Test
   void matchingMetadataFileWithoutPrefixShouldHaveExpectedName() {
     final FileBasedSigningMetadataFile fileBasedSigningMetadataFile =
-        load(NO_PREFIX_LOWERCASE_ADDRESS, KEY_FILE, PASSWORD_FILE);
+        load(NO_PREFIX_LOWERCASE_ADDRESS + CONFIG_FILE_EXTENSION, KEY_FILE, PASSWORD_FILE);
```

Comment:
nit: I'd rename NO_PREFIX_LOWERCASE_ADDRESS to just LOWER_CASE_ADDRESS (the NOP_PREFIX is a given)
...then in the next function chuck in "arbitrary_prefix" + LOWER_CASE_ADDRESS ... makes it clear right here in this test what is going on, without having to go elsewhere.